VirtoCommerce.CustomerModule.Web/Scripts/blades/member-detail.js

```diff
@@ -123,10 +124,24 @@
                 skip: $select.page * $scope.pageSize
             },
             function (data) {
-                $scope.organizations = $scope.organizations.concat(data.results);
+                joinOrganizations(data.results);
                 $select.page++;
             });
     };
 
+    function loadCustomerOrganizations() {
+        organizationsResource.getByIds(
```

Comment:
this line may cause cal API event if a customer doesn't have any organization. need to add a check and call API only when customer has any organization

Reply:
Done